packages/cli/reference/auth.md

```diff
@@ -3,88 +3,87 @@
 
 Manage your authentication profiles.
 
-* [`rockset auth:add NAME APIKEY [APISERVER]`](#rockset-authadd-name-apikey-apiserver)
-* [`rockset auth:delete NAME`](#rockset-authdelete-name)
-* [`rockset auth:list`](#rockset-authlist)
-* [`rockset auth:use NAME`](#rockset-authuse-name)
+* [`rockset auth add NAME APIKEY [APISERVER]`](#rockset-auth-add-name-apikey-apiserver)
```

Comment:
https://github.com/oclif/core/blob/HEAD/MIGRATION.md#configure-the-topicseparator I think we need to do this otherwise the command structure actually changes

Reply:
I checked and it works with either space or colon seperators